### ISSUE TITLE
Glyph class misses the 'advance' attribute

### ISSUE THREAD

**Issue by horasio** [rougier/freetype-py#3]
First: thank you for this nice python module :-)
Second: the doc string says the Glyph class should contain an attribute 'advance' but it is missing. I think it should be available just as in the GlyphSlot class. ( Currently, I have to cache it separately from the Glyph obtain via face.glyph.get_glyph() )

**Comment by adrientetar** [comment]
Was this issue ever patched? If not, it should probably remain open.

**Comment by rougier** [comment]
Good point. I nee to check, I will close it again it it has been solved.
